src/Jobs/Import/BaseEcImportJob.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Wm\WmPackage\Jobs\Import;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\DB;
+
+abstract class BaseEcImportJob extends BaseImportJob
+{
+    const DEFAULT_LON_LAT = ['POINT Z' => '12.4964 41.9028 0', 'LINESTRING Z' => '12.4964 41.9028 0, 12.5033 41.9019 0, 12.5092 41.9101 0, 12.4964 41.9028 0'];
+
+    protected function getModelKey(): string
+    {
+        return 'ec_';
+    }
+
+    protected function transformData(array $data): array
+    {
+        $transformedData = parent::transformData($data);
+
+        // if geometry is null set a default 3D geometry
+        if (empty($transformedData['geometry'])) {
+            $transformedData['geometry'] = DB::raw("ST_GeomFromText('{$this->getGeometryType()} ({$this::DEFAULT_LON_LAT[$this->getGeometryType()]})')");
```

Comment:
$trasformedData restituisce nella geometry una expression o una stringa. Propongo di fare in modo che convertTo3DGeometry restituisca un'expression (sarebbe più efficiente) e di spostare le query in GeometricComputationService